spec/models/zipped_moab_version_spec.rb

```diff
@@ -50,7 +50,7 @@
     it 'if PC is unreplicatable, returns false, does not enqueue' do
       expect(pc).to receive(:replicatable_status?).and_return(false)
       expect(ZipmakerJob).not_to receive(:perform_later)
-      expect(zmv.replicate!).to be(false)
+      expect(zmv.replicate!).to be(nil)
```

Comment:
i didn't add the analogous expectation to line 58, because we monkeypatch `ZipmakerJob.perform_later` to do nothing for most of the tests, and it didn't seem worth tweaking that or jumping through other hoops just to confirm that `ActiveJob.perform_later` returns what you'd expect.

https://github.com/sul-dlss/preservation_catalog/blob/ad124bdd0cda75ccc8e775166b10f9202dc57e26/spec/rails_helper.rb#L56-L58